RocketChat/Rocket.Chat.ReactNative
# [FIX] Keyboard auto focus on TwoFactor component

@RocketChat/ReactNative

## Proposed changes
This PR fixes the not auto focus on the TwoFactor component `TextInput`.

## How to test or reproduce
Set 2fa in your account, and login again to see the TwoFactor component with its focused `TextInput` 

## Screenshots

![ezgif com-resize (1)](https://user-images.githubusercontent.com/29265857/88101769-df3a3480-cb74-11ea-912c-bcca656bb409.gif)

## Types of changes





## Checklist




## Further comments


- Just setting `TextInput` default `autoFocus` wasn't working. After reading [this issue](https://github.com/software-mansion/react-native-screens/issues/89), the workaround seems to be delaying the `TextInput` `ref.focus()`.
- Instead of hardcoding a time on `setTimeout` I've used `runAfterInteractions` method from `InteractionManager`. [This method](https://reactnative.dev/docs/interactionmanager#runafterinteractions) schedule a function to run after all the interactions have completed, so when it finishes it'll call `ref.focus()`.
